yw_website/yw_website/apps/website/templates/includeable/workflow_list.html

```diff
@@ -26,37 +26,42 @@
 
         {% for workflow in workflow_list %}
             <li>
-                <div classs = "tags">
-                    Tags: {{ workflow.tags | join:", " }}
-                </div>
                 <div class = "title">
-                    {{ workflow.title|linebreaks }}
+                        {{ workflow.title|linebreaks }}
+                    
                 </div>
                 <div class = "last_modified"> 
                     {{ workflow.version_modified }}
                 </div>
 
-                <div id="graph{{ forloop.counter }}" class = workflow-image></div>
+                <!-- <div id="graph{{ forloop.counter }}" class = workflow-image></div> -->
 
-                <div class = "workflow-details" id="description">
+                
+                <div  class = "workflow-description" id="description">
                     <b>Description: </b>{{ workflow.description }}
                 </div>
+                <div >                
+                    <input id = "description-button" type="button" value = "Read More..." class = "expand-description-button" onClick="description_view()" ></input>
```

Comment:
id's must be unique. This element is created multiple times since it is in a for loop, so you are violating that. Change the id to `id="description-button{{ forloop.counter }}".